lib/rantly/generator.rb

```diff
@@ -141,8 +141,18 @@ def positive_integer
     range(0)
   end
 
-  def float
-    rand
+  def float(distribution=nil, params={})
+    case distribution
+    when :normal
+      params[:center] ||= 0
+      params[:scale] ||= 1
+      raise "The distribution scale should be greater than zero" unless params[:scale].positive?
+      # Sum of 6 draws from a uniform distribution give as a draw of a normal
+      # distribution centered in 3 (central limit theorem).
+      ([rand, rand, rand, rand, rand, rand].sum - 3) * params[:scale] + params[:center]
```

Comment:
[Enumerable#sum](http://ruby-doc.org/core-2.4.0/Enumerable.html#method-i-sum) was added in ruby 2.4.0. Since ruby 2.3 is still being maintained (see [here](https://www.ruby-lang.org/en/downloads/branches/)) we should widen support by using `reduce`, e.g.:

```ruby
[...].reduce(0, &:+)
```

Reply:
Actually in Travis there are many Ruby versions: https://travis-ci.org/rantly-rb/rantly/builds/320015060

Si I guess we want to support all of them. I'll change this then. 😉